package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "webtorrent-desktop",
   "description": "WebTorrent, the streaming torrent client. For Mac, Windows, and Linux.",
-  "version": "0.19.0",
+  "version": "0.19.1",
```

Comment:
Never bump versions in a PR, this is done when releases are about to be done.

Reply:
Ouch, I'm too used to that.
I'll update it.
Thanks!